bin/generate-release-notes.js

```diff
@@ -0,0 +1,96 @@
+'use strict'
+
+const fs = require('fs')
+const github = require('./github')
+
+const FILE_NAME = 'NEWS.md'
+const PROPOSED_NOTES_HEADER = 'Proposed Release Notes'
+const NEXT_VERSION_HEADER = '### vNext (TBD):'
+
+async function generateReleaseNotes() {
+  try {
+    const latestRelease = await github.getLatestRelease()
+    console.log(`The latest release is: ${latestRelease.name} published: ${latestRelease.published_at}`)
+    console.log(`Tag: ${latestRelease.tag_name}, Target: ${latestRelease.target_commitish}`)
+
+    const tag = await github.getTagByName(latestRelease.tag_name)
+    console.log('The tag commit sha is: ', tag.commit.sha)
+
+    const commit = await github.getCommit(tag.commit.sha)
+    const commitDate = commit.commit.committer.date
+
+    console.log(`Finding merged pull requests since: ${commitDate}`)
+
+    const mergedPullRequests = await github.getMergedPullRequestsSince(commitDate)
+    console.log(`Found ${mergedPullRequests.length}`)
+
+    const releaseNoteData = mergedPullRequests.map((pr) => {
+      const parts = pr.body.split(/(?:^|\n)##\s*/g)
+      const {1: proposedReleaseNotes} = parts
```

Comment:
Is there a chance that the 2nd part of the parts array may not exist?

Reply:
It is certainly possible this would blow up with a malformed PR template. Was just aiming for golden path here.

We can tackle more robustness as we iterate.